src/index.js

```diff
@@ -0,0 +1,269 @@
+import './style.scss';
+import image from './img/backgroung.jpg';
+// DOM manipulation
+let addNewList = document.getElementById("addNewList");
+let addNewTodo = document.getElementById("addNewTodo");
+let newListSaveBtn = document.getElementById("newListSaveBtn");
+let listGroup = document.getElementById("listName");
+let linkList = document.getElementById("link-list");
+let btns = [addNewList, addNewTodo];
+let currentGroupId = "";
+let currentTodoId = null;
+const saveBtn = document.querySelector(".save-btn");
+const cancelBtn = document.querySelectorAll(".cancel-btn");
+const todoForm = document.getElementById("todo-form");
+const newToDo = document.getElementsByClassName('new-todo')[0];
+const todoBox = document.getElementById("todo-box");
+
+addNewList.addEventListener('click', e => {
+    document.getElementsByClassName('new-list')[0].style.display = "block";
+})
+
+addNewTodo.addEventListener('click', e => {
+    newToDo.style.display = 'block';
+})
+
+
+for (let btn of cancelBtn) {
+    btn.addEventListener('click', () => {
+        document.getElementsByClassName('new-list')[0].style.display = "none";
+        newToDo.style.display = 'none';
+    })
+}
+
+
+
+// Creating Groups and Group Logic
+
+let groups = {
+    1: {
+        id: 1,
+        title: "My Default List",
+        description: "Random task to complete",
+        todos: {
+            12: {
+                id: 12,
+                title: "swim",
+                description: "Buy Swimsuit",
+                dueDate: new Date(),
+                priority: "High",
+                completed: false
+            },
+            13: {
+                id: 13,
+                title: "run",
+                description: "Buy Swimsuit",
+                dueDate: new Date(),
+                priority: "High",
+                completed: true
+            },
+        }
+    }
+
+}
+
+
+
+const generateId = () => {
+    const generatedNumber = Math.floor(Math.random() * 100);
```

Comment:
Let separate the DOM from application logic, you can create another file for DOM.

![image](https://user-images.githubusercontent.com/31897434/73420341-70e18b80-4322-11ea-840a-fa14e04b592b.png)

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622602/